pkg/volume/nfs/nfs.go

```diff
@@ -194,15 +194,15 @@ func (nfsMounter *nfsMounter) CanMount() error {
 	exec := nfsMounter.plugin.host.GetExec(nfsMounter.plugin.GetPluginName())
 	switch runtime.GOOS {
 	case "linux":
-		if _, err := exec.Run("/bin/ls", "/sbin/mount.nfs"); err != nil {
+		if _, err := exec.Run("test", "-x", "/sbin/mount.nfs"); err != nil {
```

Comment:
Can't we just do os.Stat("/path/to/whatever") ?

Reply:
No, `mount.nfs` can be in a container. `exec.Run` runs the command in the right place.